pkg/assetfetcher/http_test.go

```diff
@@ -545,7 +546,7 @@ func TestFetchAssetsAssetPayloadToAssetEventError(t *testing.T) {
 	mockRT := NewMockRoundTripper(ctrl)
 
 	resp := SiteAssetsResponse{
-		Resources: []Asset{{History: assetHistoryEvents{AssetHistory{Type: "SCAN", Date: "not a time"}}}},
+		Resources: []Asset{{History: assetHistoryEvents{AssetHistory{Type: "SCAN", Date: "2019-05-14T15:03:47.000Z"}}}},
```

Comment:
Question: I'm not seeing why this change was made. Why does this asset fail to convert?

Reply:
I added a check before AssetPayloadtoAssetEvent() to validate the asset history, so in order to trigger AssetPayloadtoAssetEvent(), I needed to change the Date field to be something valid.